backend.native/tests/interop/j2objc/main.kt

```diff
@@ -0,0 +1,17 @@
+import kotlinx.cinterop.*
+import kotlin.test.*
+
+fun main(args: Array<String>) {
+  autoreleasepool {
+    testMethods()
+  }
+}
+
+private fun testMethods() {
+  val myObject = j2objctest.Foo()
+  assertEquals(100, myObject.return100())
+  assertEquals(43, myObject.returnNum(43))
+  assertEquals(47, myObject.add2(16,31))
+
+  assertEquals(100, j2objctest.Foo.return100Static())
+}
```

Comment:
Maybe rename this to systemTest.kt?

Reply:
it might be best to follow convention elsewhere for this.

Reply:
Checked, seems like it's usually main.kt if it's for a multiple file test.